lib/requests/meal_requests.js

```diff
@@ -59,8 +60,46 @@ function getRemainingCalories(total, name) {
      $(`#${name}-table td.remaining-calories`).text(-(total - 800));
   } else {
      return "no data"}
-}
+  }
+
+  var deleteFood = function(e) {
+    e.preventDefault();
+    var deleteFoodId = this.id.search(/\d/);
+    var tableName = $(this).parents()[2].id
+
+   function mealId(name) {
+     if(name === "Breakfast-table") {
+      return ("1")
```

Comment:
Looks good, we might want to try an array or object like she showed in class on thursday for the refactor.